components/NavigationBar/README.md

```diff
@@ -70,6 +70,10 @@ Read the button bar section on
 [UIBarButtonItem properties](../ButtonBar/#uibarbuttonitem-properties) to learn more about
 supported UIBarButtonItem properties.
 
+Note: The UIBarButtonItem instances set on MDCNavigationBar cannot be used to specify the popover's
+anchor point in UIPopoverPresentationController. The sourceView and sourceRect on
```

Comment:
Tip: Please provide a small snippet showing how users should use that API instead.

Reply:
Addressed in 1149514bf55c01936ccb00748a4b5b6ffcc5e552